packages/fx-core/src/common/samples-config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/v0.1.0/",
-    "defaultPackageLink": "https://github.com/OfficeDev/TeamsFx-Samples/archive/refs/tags/v0.1.0.zip",
+    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/ga/",
```

Comment:
ne ga tag? or should it be another sem-version tag?

Reply:
gallery in ga branch is as same as gallery on dev branch.
since tag always hold on the release branch, with head we can test on the top of ga branch. 
otherwise with tag, we need to update the tag.
once the ga branch merge to main, then tag on the branch.
so I think tag head of ga branch, is convenient.

Reply:
sure, will change to tag